integrated_channels/integrated_channel/models.py

```diff
@@ -401,7 +401,28 @@ class ContentMetadataItemTransmission(TimeStampedModel):
         null=True,
     )
     deleted_at = models.DateTimeField(
-        help_text='Date when the content transmission was deleted',
+        help_text='Date when the content transmission was deleted in the remote API',
+        blank=True,
+        null=True
+    )
+    created_at = models.DateTimeField(
+        help_text='Date when the content transmission was created in the remote API',
+        blank=True,
+        null=True
+    )
+    updated_at = models.DateTimeField(
+        help_text='Date when the content transmission was last updated in the remote API',
+        blank=True,
+        null=True
+    )
+    api_response_status_code = models.CharField(
+        help_text='The most recent remote API call response HTTP status code'
+        max_length=100,
+        blank=True,
+        null=True
+    )
+    api_response_body = models.TextField(
```

Comment:
i wanted more descriptive names than `status`. i also want to store all response bodies, not just error response bodies.